airbyte-cdk/python/README.md

```diff
@@ -152,8 +159,8 @@ HTTP requests to `localhost:8113/data` should now return the body defined in the
 
 ## Coming Soon
 
-* Full OAuth 2.0 support \(including refresh token issuing flow via UI or CLI\)
-* Airbyte Java HTTP CDK
-* CDK for Async HTTP endpoints \(request-poll-wait style endpoints\)
-* CDK for other protocols
-* Don't see a feature you need? [Create an issue and let us know how we can help!](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=type%2Fenhancement&template=feature-request.md&title=)
+- Full OAuth 2.0 support \(including refresh token issuing flow via UI or CLI\)
+- Airbyte Java HTTP CDK
+- CDK for Async HTTP endpoints \(request-poll-wait style endpoints\)
+- CDK for other protocols
+- Don't see a feature you need? [Create an issue and let us know how we can help!](https://github.com/airbytehq/airbyte/issues/new?assignees=&labels=type%2Fenhancement&template=feature-request.md&title=)
```

Comment:
```suggestion
- Don't see a feature you need? [Create an issue and let us know how we can help!](https://github.com/airbytehq/airbyte/discussions/categories/ideas-and-features)
```

We're trying to push people to create Discussion against Issues for features (clear upvote system)

Reply:
I think I even seen that in _one_ spot it already leads to discussions! Good catch, I'll clean this up.